Cargo.toml

```diff
@@ -527,7 +541,7 @@ nix = { workspace = true, features = ["process", "signal", "user", "term"] }
 rlimit = "0.10.1"
 xattr = { workspace = true }
 
-# Specifically used in test_uptime::test_uptime_with_file_containing_valid_boot_time_utmpx_record
+# Specifically, used in test_uptime::test_uptime_with_file_containing_valid_boot_time_utmpx_record
```

Comment:
I see why you would add a comma here, but I think that's not quite right. It should be read more as in "Used specifically in ..."